[2025-03-17T19:01:36.181+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T19:01:36.498+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.pages_project_a_db scheduled__2025-03-17T18:00:00+00:00 [queued]>
[2025-03-17T19:01:36.559+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.pages_project_a_db scheduled__2025-03-17T18:00:00+00:00 [queued]>
[2025-03-17T19:01:36.578+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T19:01:36.771+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.pages_project_a_db> on 2025-03-17 18:00:00+00:00
[2025-03-17T19:01:36.822+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2428) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T19:01:36.831+0000] {standard_task_runner.py:72} INFO - Started process 2447 to run task
[2025-03-17T19:01:37.009+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.pages_project_a_db', 'scheduled__2025-03-17T18:00:00+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpbf7ahlxy']
[2025-03-17T19:01:37.069+0000] {standard_task_runner.py:105} INFO - Job 157: Subtask load_tables.pages_project_a_db
[2025-03-17T19:01:37.848+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.pages_project_a_db scheduled__2025-03-17T18:00:00+00:00 [running]> on host c032966e105e
[2025-03-17T19:01:38.869+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.pages_project_a_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-17T18:00:00+00:00'
[2025-03-17T19:01:38.871+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T19:01:39.262+0000] {sql.py:278} INFO - Executing: select load_tables('pages_project_a_db', 'pages');
[2025-03-17T19:01:39.560+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T19:01:39.638+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T19:01:39.660+0000] {sql.py:553} INFO - Running statement: select load_tables('pages_project_a_db', 'pages');, parameters: None
[2025-03-17T19:01:39.788+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T19:01:40.052+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T19:01:40.070+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.pages_project_a_db, run_id=scheduled__2025-03-17T18:00:00+00:00, execution_date=20250317T180000, start_date=20250317T190136, end_date=20250317T190140
[2025-03-17T19:01:40.361+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T19:01:40.949+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T19:01:41.030+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
